common/src/Model/Brush.cpp

```diff
@@ -916,7 +922,11 @@ namespace TrenchBroom {
          ok     - This case is always allowed, unless the brush becomes invalid, i.e., not a polyhedron.
          no     - This case is always forbidden.
          invert - This case is handled by swapping the remaining and the moving fragments and inverting the delta. This takes us from a cell at (column, row) to the cell at (row, column).
-         check  - Check whether any of the moved vertices would travel into or through the remaining fragment, or vice versa if inverted case. Also check whether the brush would become invalid, i.e., not a polyhedron.
+         check  - Check whether any of the moved vertices would travel through the remaining fragment, or vice versa if inverted case. Also check whether the brush would become invalid, i.e., not a polyhedron.
+         
+         If `allowVertexRemoval` is false, moving a face in a way that would flip its normal will return `false`
+         If `allowVertexRemoval` is true, vertices can be moved inside a remaining polyhedron.
```

Comment:
Note that I don't intend to allow the user to remove vertices in any way other than by moving vertices. Moving edges and faces should not result in a loss of vertices at all. At least that was my intention when I did this - maybe you think that we can allow more freedom here? I thought that often it would be inconvenient to allow this since you'll quickly remove vertices by accident the more vertices are being moved at the same time.